### PR TITLE
add virtual-kubelet provider for armada to LFX ideas

### PR DESCRIPTION
**Pull request by kannon92**
Project idea for Armada for Fall LFX Mentorship

### CODE REVIEW

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Thanks for this @kannon92!

It looks like part of the description is missing. Would you mind updating it?

File: programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md [hunk 1]
```diff
@@ -18,4 +18,16 @@
 
 ## Proposed Project ideas
 
+### Armada
+
+#### Build a virtual-kubelet provider for Armada
+
+- Description: (Virtual Kubelet)[https://github.com/virtual-kubelet/virtual-kubelet] is a way to bring a Kubernetes front end and allow for your execution environment to be something other than Kubernetes. We want to add a virtual-kubelet provider so we can allow for
```
Comment (RohanSasne) [inline programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:25]:
```suggestion
- Description: [Virtual Kubelet](https://github.com/virtual-kubelet/virtual-kubelet) is a way to bring a Kubernetes front end and allow for your execution environment to be something other than Kubernetes. We want to add a virtual-kubelet provider so we can allow for
```

Comment (RohanSasne) [reply programs/lfx-mentorship/2023/03-Sep-Nov/project_ideas.md:25]: Little quick fix over here 😁

**Review by nate-double-u — APPROVED** [review]
Thanks a lot for this @kannon92!

**Pull request merged**